### PR TITLE
fixing code of conduct link

### PR DESCRIPTION
**Pull request by vsoch**
Signed-off-by: Vanessa Sochat <vsochat@stanford.edu>

@cyphar is this the correct formatting? I'll need to fix a few other repos as well!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The Go types and validation should be compatible with the current Go release; ea
 
 Additional documentation about how this group operates:
 
-- [Code of Conduct](code-of-conduct)
+- [code-of-conduct]
```
Comment (vbatts) [inline README.md]: `[Code of Conduct][code-of-conduct]`

Comment (vsoch) [reply README.md]: Just fixed!

**Comment by vbatts** [comment]
LGTM

**Comment by cyphar** [comment]
You should squash your commits.

**Comment by vsoch** [comment]
Squashed!

**Comment by cyphar** [comment]
There's a conflict, you'll need to rebase.

**Comment by vsoch** [comment]
I'll try rebase in a few minutes.

**Comment by vsoch** [comment]
Ahhh I messed it up.

**Comment by vsoch** [comment]
This is like learning how to snow board *falls on face* going to try again!

**Pull request closed without merging**